lib/semian.rb

```diff
@@ -127,6 +127,12 @@ def to_s
   # is calculated as (workers * quota).ceil
   # Mutually exclusive with the 'ticket' argument.
   #
+  # +min_tickets+: When using the quota strategy, never allocated fewer tickets than this value.
+  # This is particularly useful during deploys, where the number of tickets will be constrained
+  # due to the number of workers going offline. To prevent spurious timeouts, you may set a minimum
+  # ticket count. The resulting ticket count will be max(min_ticket_count, calculated_quota_tickets).
+  # This option is ignored unless you are using the quota strategy.
```

Comment:
Should we call this `quota_min_tickets` to make extra certain it's only relevant for that strategy? It you think it's better, go ahead. Do you want to raise if `min_tickets` is set to `> 0` and `tickets` is set directly as well?